packages/shared-ux/code_editor/impl/code_editor.tsx

```diff
@@ -375,6 +381,12 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
           monaco.languages.registerCodeActionProvider(languageId, codeActions);
         }
       });
+
+      monaco.editor.addKeybindingRule({
+        // eslint-disable-next-line no-bitwise
+        keybinding: monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyF,
+        command: enableFindAction ? 'actions.find' : null,
```

Comment:
nit: instead of registering the keybindings with a null action, I think we could only register the key binding if the find action is enabled

Reply:
Since we added the find action import in `packages/kbn-monaco/src/monaco_imports.ts`, this action is now enabled by default in every Monaco editor, unless we disable it. Therefore, we need to specify explicitly that the key binding should run no action; otherwise it would run the find action by default.